ui/src/modules/jobs/components/StepIndicator.tsx

```diff
@@ -5,44 +5,63 @@ const StepIndicator: React.FC<StepIndicatorProps> = ({
 	step,
 	index,
 	currentStep,
+	onStepClick,
+	isEditMode,
 }) => {
 	const isActive = steps.indexOf(currentStep) >= index
```

Comment:
can we extract it out to const, we're using multiple times? `steps.indexOf(currentStep)`
```suggestion
	const isActive = steps.indexOf(currentStep) >= index
```

Reply:
moved to top as constant